### DISCUSSION TITLE
I swear the NTM discord is absolute cancer

### DISCUSSION THREAD

**Discussion by DB4000K** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1161] [General]
Do NOT join the discord. IT IS A TRAP. THEY WANT YOUR FLESH.

**Comment by LePeeperSauvage** [comment]
We hear you loud and clear, your complaints have been recorded, and assigned to the related units. Estimated delay time : [Boo-Womp].
We thank you for your patience!

**Reply by DB4000K** [reply]
Damn that is a long time, I don't know if I can wait
